UnityProject/Assets/Scripts/Closets/ClosetControl.cs

```diff
@@ -402,15 +466,9 @@ public bool WillInteract(HandApply interaction, NetworkSide side)
 	public void ServerPerformInteraction(HandApply interaction)
 	{
 		// Is the player trying to put something in the closet?
-		if (interaction.HandObject != null)
+		if (Validations.HasItemTrait(interaction.HandObject, CommonTraits.Instance.Emag))
```

Comment:
does this not throw an NRE when interacting with empty hand?

Reply:
It doesn't throw any errors. I copied this method off the Doors script.

Reply:
aight, great